pkgs/games/hex-a-hop/default.nix

```
@@ -0,0 +1,41 @@
+{lib, stdenv, makeDesktopItem, copyDesktopItems, fetchurl, SDL, SDL_mixer, SDL_ttf}:
+
+stdenv.mkDerivation rec {
+   pname = "hex-a-hop";
+   version = "1.1.0";
+
+   src = fetchurl {
+     url = "https://downloads.sourceforge.net/project/hexahop/1.1.0/hex-a-hop-1.1.0.tar.gz";
+     sha256 = "sha256-gL8Z2TaoQwyrW8Ro7mgnw4BS4CbGCet+zhYxfnkyoNY=";
+   };
+
+   buildInputs = [ SDL SDL_mixer SDL_ttf copyDesktopItems];
+
+   makeFlags = [ "PREFIX=$(out)" ];
+
+   desktopItems = makeDesktopItem rec {
+     name = pname;
+     exec = name;
+     icon = name;
+     desktopName = name;
```

This is simpler and more clear:

```suggestion
   desktopItems = makeDesktopItem {
     name = pname;
     exec = pname;
     icon = pname;
     desktopName = pname;
```

`name` is confusing since that can be a drv attr with a very different meaning.